tests/acceptance/bootstrap/FeatureContext.php

```diff
@@ -345,6 +344,15 @@ public function aSprintExistsForTheProjectInPhabricatorButNotInPhragile($sprintT
 	}
 
 	/**
+	 * @Given I copied the :sprint Phabricator ID from Phabricator
+	 */
+	public function iCopiedThePhabricatorIdFromPhabricator($sprint)
+	{
+		$phabricatorProject = $this->getOrCreatePhabricatorProjectFromTitle($sprint);
+		$this->phabricatorProjectID = $phabricatorProject['id'];
+	}
+
+	/**
 	 * @Given I copied the :project :sprint Phabricator ID
 	 */
 	public function iCopiedThePhabricatorId($project, $sprint)
```

Comment:
doesn't adding that `iCopiedThePhabricatorIdFromPhabricator` method make this `iCopiedThePhabricatorId` redundant?
It seems like the latter has only been used in the `connect_existing_project.feature` and has been replaced by the new method.